projects/ngrx.io/content/guide/data/index.md

```diff
@@ -0,0 +1,143 @@
+# NgRx Data
+
+NgRx Data is an extension that offers a gentle introduction to NgRx by simplifying management of **entity data** while reducing the amount of boilerplate.  
```

Comment:
```suggestion
NgRx Data is an extension that offers a gentle introduction to NgRx by simplifying management of **entity data** while reducing the amount of explicitness.  
```

Reply:
Fixed in 5ef16b0.

Reply:
<insert @brandonroberts boilerplate yelling boilerplate .gif> 😂